### PR TITLE
Flagship: meta content

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## What does this change?

Installs a few changes in how meta is laid out for the flagship. There is a second part to this work for desktop, which I'll do in a separate PR.

## Screenshots

Just trust me 😆

### CODE REVIEW

File: applications/app/views/fragments/audioBody.scala.html [hunk 1]
```diff
@@ -63,13 +54,13 @@
...
-                         </div>
-                     }
+                    @* 
+                    @*  @if(audio.fields.body.nonEmpty) {
+                    @*      <div class="content__secondary-column content__secondary-column--media content__secondary-column--audio hide-on-childrens-books-site"
+                    @*      aria-hidden="true">
+                    @*          @fragments.commercial.standardAd("right", Seq("dark"), Map("desktop" -> Seq("300,250", "300,274", "fluid")))
+                    @*      </div>
+                    @*  } *@
                 </div>
```
Comment (ajwl) [inline applications/app/views/fragments/audioBody.scala.html]: Do you want to keep this commented out code or can it be deleted?

Comment (regiskuckaertz) [reply applications/app/views/fragments/audioBody.scala.html]: Yeah it has to stay, I'll put it back in in the next iteration

Comment (ajwl) [reply applications/app/views/fragments/audioBody.scala.html]: Fair enough

**Comment by ajwl** [comment]
👍

**Pull request merged**